# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: parser/html/nsHtml5DocumentBuilder.cpp
# Commit: 77388b0795b4
# Full Hash: 77388b0795b4f9a30171c922d6b649c52e5f6c87
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-23 09:34:36
# Description:
#   Bug 1745142 - Communicate encoding commitment via speculative load queue. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133996
# ==============================================================================

diff -r a388127095db -r 77388b0795b4 parser/html/nsHtml5DocumentBuilder.cpp
--- a/parser/html/nsHtml5DocumentBuilder.cpp	Wed Dec 22 15:47:08 2021 +0000
+++ b/parser/html/nsHtml5DocumentBuilder.cpp	Wed Dec 22 15:54:49 2021 +0000
@@ -39,14 +39,6 @@
   return aReason;
 }
 
-void nsHtml5DocumentBuilder::SetDocumentCharsetAndSource(
-    NotNull<const Encoding*> aEncoding, int32_t aCharsetSource) {
-  if (mDocument) {
-    mDocument->SetDocumentCharacterSetSource(aCharsetSource);
-    mDocument->SetDocumentCharacterSet(aEncoding);
-  }
-}
-
 void nsHtml5DocumentBuilder::UpdateStyleSheet(nsIContent* aElement) {
   auto* linkStyle = LinkStyle::FromNode(*aElement);
   if (!linkStyle) {